script/proto-docs.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -eux -o pipefail
+
+mkdir -p docs/grpc
```

Comment:
should this be under $DOCROOT ?

Reply:
nope, `$DOCROOT` is for the base path to use in the generated docs, not the source path. I guess `DOCROOT` should be renamed to make that clear?